# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/RtcpEventObserver.h
# Commit: 0552c0b4a079
# Full Hash: 0552c0b4a079a695edfca100df68c4ad82481e58
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Refactor away RtcpEventObserver in favor of MediaEvents. r=bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123223
# ==============================================================================

diff -r 877a0fcc6da9 -r 0552c0b4a079 dom/media/webrtc/libwebrtcglue/RtcpEventObserver.h
--- a/dom/media/webrtc/libwebrtcglue/RtcpEventObserver.h	Mon Aug 16 15:57:10 2021 +0200
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,20 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef RTCP_EVENT_OBSERVER_
-#define RTCP_EVENT_OBSERVER_
-
-namespace mozilla {
-/**
- * This provides an interface to allow for receiving notifications
- * of rtcp bye packets and timeouts.
- */
-class RtcpEventObserver {
- public:
-  virtual void OnRtcpBye() = 0;
-  virtual void OnRtcpTimeout() = 0;
-};
-
-}  // namespace mozilla
-#endif